infinispan-remote/src/test/java/org/hibernate/ogm/datastore/infinispanremote/test/storedprocedures/InfinispanNamedParametersStoredProcedureCallTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.datastore.infinispanremote.test.storedprocedures;
+
+import javax.persistence.EntityManager;
+import javax.persistence.ParameterMode;
+import javax.persistence.PersistenceException;
+import javax.persistence.StoredProcedureQuery;
+
+import org.hibernate.ogm.backendtck.storedprocedures.Car;
+import org.hibernate.ogm.utils.TestForIssue;
+import org.hibernate.ogm.utils.jpa.OgmJpaTestCase;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+/**
+ * Testing call of stored procedures
+ *
+ * @author The Viet Nguyen &amp;ntviet18@gmail.com&amp;
+ */
+@TestForIssue(jiraKey = { "OGM-1430" })
+public class InfinispanNamedParametersStoredProcedureCallTest extends OgmJpaTestCase {
+
+	@Rule
+	public ExpectedException thrown = ExpectedException.none();
+
+	protected EntityManager em;
+
+	@Before
+	public void setUp() {
+		em = getFactory().createEntityManager();
+	}
+
+	@After
+	public void tearDown() {
+		em.close();
+	}
+
+	@Test
+	public void testExceptionWhenUsePositionalParameters() throws Exception {
+		thrown.expect( PersistenceException.class );
+		thrown.expectMessage( "org.hibernate.HibernateException: OGM000094" );
+		StoredProcedureQuery storedProcedureQuery = em.createStoredProcedureQuery( Car.SIMPLE_VALUE_PROC, Car.class );
+		storedProcedureQuery.registerStoredProcedureParameter( 0, Integer.class, ParameterMode.IN );
+		storedProcedureQuery.setParameter( 0, 1 );
+		storedProcedureQuery.getSingleResult();
```

Comment:
In OgmJpaTestCase there is a method called `inTransaction` that you can use like this:

```
inTransaction( em -> {
    StoredProcedureQuery storedProcedureQuery = em.createStoredProcedureQuery( Car.SIMPLE_VALUE_PROC, Car.class );
    storedProcedureQuery.registerStoredProcedureParameter( 0, Integer.class, ParameterMode.IN ); 
    storedProcedureQuery.setParameter( 0, 1 );
    storedProcedureQuery.getSingleResult();
} );
```

We are trying to use it in all the new tests and this way you can remove the setup and tear down methods